.github/workflows/test.yml

```diff
@@ -6,10 +6,21 @@ on:
 jobs:
   test:
     runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python: ["3.6", "3.7", "3.8", "3.9"]
     steps:
       - uses: actions/checkout@v2
         with: { submodules: recursive }
+      - uses: conda-incubator/setup-miniconda@v2
+        with: { mamba-version: "*", channels: "conda-forge", channel-priority: true }
       - name: install dependencies
         shell: bash -l {0}
         run: |
-          echo Hello World.
+          mamba env update --quiet -n test -f environment.yml
+          conda list
+      - name: cli-test
+        shell: bash -l {0}
+        run: |
+          python -m svgdigitizer digitize test/data/Ni111_NaOH_Beden1985_Fig2c
+          diff test/data/Ni111_NaOH_Beden1985_Fig2c.csv test/data/Ni111_NaOH_Beden1985_Fig2c.csv.expected
```

Comment:
For every Pull Request, we will run this check, i.e., run the digitizer on a sample file and check that the generated CSV matches the expected output. (This is not a great check but it was the easiest check that I could think of now.)

Reply:
You can see the result of these tests at the bottom of this Pull Request where it (hopefully) says "All checks have passed".